HeliPort/Credits.rtf

```diff
@@ -66,6 +66,8 @@ Communication with\'a0{\field{\*\fldinst{HYPERLINK "https://github.com/zxystd/it
 
 \f1\b \cf2 License\
 \
+\pard\pardeftab720\partightenfactor0
 
-\f0\b0 BSD-3-Clause\
-}
\ No newline at end of file
+\f0\b0 \cf5 \cb1 This software is distributed through the 3-Clause BSD License, otherwise known as New BSD License. \
```

Comment:
you can simply remove `\cf5` and `\cb1` here, which means foreground & background color selection, then it will inherit the color set above.